pkgs/applications/science/logic/cvc5/default.nix

```diff
@@ -1,39 +1,101 @@
-{ lib, stdenv, fetchFromGitHub, pkg-config, cmake, flex, cadical, symfpu, gmp, python3, gtest, libantlr3c, antlr3_4, boost, jdk }:
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  pkg-config,
+  cmake,
+  flex,
+  cadical,
+  symfpu,
+  gmp,
+  python3,
+  gtest,
+  libantlr3c,
+  antlr3_4,
+  boost,
+  jdk,
+  withPythonBindings ? false,
+}:
 
 stdenv.mkDerivation rec {
   pname = "cvc5";
   version = "1.1.2";
 
   src = fetchFromGitHub {
-    owner  = "cvc5";
-    repo   = "cvc5";
-    rev    = "cvc5-${version}";
-    hash  = "sha256-v+3/2IUslQOySxFDYgTBWJIDnyjbU2RPdpfLcIkEtgQ=";
+    owner = "cvc5";
+    repo = "cvc5";
+    rev = "cvc5-${version}";
+    hash = "sha256-v+3/2IUslQOySxFDYgTBWJIDnyjbU2RPdpfLcIkEtgQ=";
   };
 
-  nativeBuildInputs = [ pkg-config cmake flex ];
+  pythonic = fetchFromGitHub {
+    owner = "cvc5";
+    repo = "cvc5_pythonic_api";
+    rev = "0bb7f092113e3b933dcd61b0959f3e9be9638109";
```

Comment:
How do you propose to manage the version of this?

I suggest trying to synchronize it from the version specified in https://github.com/cvc5/cvc5/blob/887ecb98a2dd682c19888cde3789871b3dc158fe/cmake/FindCVC5PythonicAPI.cmake#L37. Actually _extracting_ it from there (thus daisy-chaining the two `fetchFromGitHub`s somehow) would be quite painful, but you might be able to add a line that checks the fetched version corresponds with the version expected in `FindCVC5PythonicAPI.cmake` and abort if not, prompting the updater to bump the fetched version.